pg_search/src/index/reader/index.rs

```diff
@@ -189,11 +189,11 @@ pub struct SearchIndexReader {
     underlying_reader: IndexReader,
     underlying_index: Index,
 
-    // [`Buffer`] has a Drop impl, so we hold onto the lock, but don't otherwise use it
+    // [`PinnedBuffer`] has a Drop impl, so we hold onto the lock, but don't otherwise use it
```

Comment:
Since this type of buffer does not hold a lock for the duration of it being pinned, can update this comment accordingly.

```suggestion
    // [`PinnedBuffer`] has a Drop impl to release the underlying pinned buffer, so we hold onto it but don't otherwise use it
```

Reply:
done